core.coffee

```diff
@@ -434,6 +446,34 @@
 						url += '?' + queryOptions.join('&')
 					return url
 
+			compileBatch: (data) ->
+				if !utils.isArray(data)
+					throw new Error('Batch must be passed an array')
+				data = utils.cloneDeep(data)
+
+				buildReq = (req) =>
+					# This is a normal request
+					if req.url?
+						req['Content-Transfer-Encoding'] = 'binary'
+						if req.body?
+							req['Content-Type'] = 'application/json'
+							b = JSON.stringify(req.body)
+						else
+							req['Content-Type'] = 'application/http'
+							b = ''
+						req.body = req.method + ' ' + @compile(req) + ' HTTP/1.1' + '\r\n' + b
```

Comment:
It's very minor, but this can be 
```
req.body = req.method + ' ' + @compile(req) + ' HTTP/1.1\r\n' + b
```
but really I think you should be using string interpolation